tutor/jobs.py

```diff
@@ -126,11 +126,18 @@ def set_theme(
         return
     python_code = "from django.contrib.sites.models import Site"
     for domain_name in domain_names:
+        if len(domain_name) > 50:
```

Comment:
Instead of 50. Use `Site._meta.get_field('name').max_length`

Shouldn't this be part of the python

Reply:
@keithgg This will require us to have django installed locally.

Reply:
Ah, is it possible to rather move it to the `python_code` block instead?

Reply:
To print warnings we are using `fmt.echo_alert` which is part of tutor and not edx-platform, which is the reason I had to add it outside.

Reply:
Cool. Let's leave it as is. :+1: